### ISSUE TITLE
HasDecoder instance for ElmConstructor is partial

### ISSUE THREAD

**Issue by mitchellwrosen** [krisajenkins/elm-export#32]
Hi, I don't know much about Elm yet, but I put something together that hit an non-exhaustive pattern match panic - there's no `MultipleConstructors` pattern for `render`ing an `ElmConstructor`.

Does this have to do with #6?

**Comment by FPtje** [comment]
Yes, it does. Both encoders and decoders are not supported for ADTs with multiple constructors for the reasons mentioned in #6.
